tilecloud_chain/wmts_get_capabilities.jinja

```diff
@@ -5,9 +5,14 @@
     xmlns:xlink="http://www.w3.org/1999/xlink"
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xmlns:gml="http://www.opengis.net/gml"
-    xsi:schemaLocation="http://schemas.opengis.net/wmts/1.0/wmtsGetCapabilities_response.xsd">
-  <ows:ServiceIdentification> </ows:ServiceIdentification>
-  <ows:ServiceProvider> </ows:ServiceProvider>
+    xsi:schemaLocation="http://schemas.opengis.net/wmts/1.0/wmtsGetCapabilities_response.xsd">{%
+  if title %}
+  <ows:ServiceIdentification>
+    <ows:Title>{{title}}</ows:Title>
+    <ows:ServiceType>OGC WMTS</ows:ServiceType>
+    <ows:ServiceTypeVersion>1.0.0</ows:ServiceTypeVersion>
```

Comment:
Shouldn't we also add
- ows:Abstract
- ows:Keywords (with ows:Keyword)
- ows:Fees
- ows:AccessConstraints

?

Full example:
```xml
<ows:Abstract>
Example service that constrains some world layers in the
GlobalCRS84Pixel Well-known scale set
</ows:Abstract>
<ows:Keywords>
  <ows:Keyword>World</ows:Keyword>
  <ows:Keyword>Global</ows:Keyword>
  <ows:Keyword>Digital Elevation Model</ows:Keyword>
  <ows:Keyword>Administrative Boundaries</ows:Keyword>
</ows:Keywords>
<ows:Fees>none</ows:Fees>
<ows:AccessConstraints>none</ows:AccessConstraints>
```

Reply:
Those attributes are optional. I have only added the mandatory ones.